src/CollageImage.js

```diff
@@ -71,9 +71,10 @@ class CollageImage extends React.Component {
 
     this._panResponder = PanResponder.create({
       onStartShouldSetPanResponder: (evt, gestureState) => true,
-      onStartShouldSetPanResponderCapture: (evt, gestureState) => true,
       onMoveShouldSetPanResponder: (evt, gestureState) => true,
-      onMoveShouldSetPanResponderCapture: (evt, gestureState) => true,
+      onMoveShouldSetPanResponderCapture: (evt, gestureState) => {
+        return Math.abs(gestureState.dx) > 5;
+      },
```

Comment:
I propose we keep this as it was and handle the tap in `onPanResponderEnd`

```suggestion
      onMoveShouldSetPanResponderCapture: (evt, gestureState) => true,
```